polaris-react/src/components/Card/Card.tsx

```diff
@@ -62,16 +63,29 @@ export const Card = ({
 
   return (
     <WithinContentContext.Provider value>
-      <Box
-        background={background}
-        padding={finalPadding}
-        shadow={polarisSummerEditions2023 ? 'card-sm-experimental' : 'md'}
-        borderRadius={hasBorderRadius ? defaultBorderRadius : undefined}
-        overflowX="hidden"
-        overflowY="hidden"
-      >
-        {children}
-      </Box>
+      {polarisSummerEditions2023 ? (
+        <ShadowBevel boxShadow="xs" borderRadius="3">
+          <Box
+            background={background}
+            padding={finalPadding}
+            overflowX="hidden"
+            overflowY="hidden"
+          >
+            {children}
+          </Box>
+        </ShadowBevel>
```

Comment:
This doesn't appear to account for the `roundedAbove` prop. I wonder if something like this would work:

```jsx
<ShadowBevel boxShadow="xs" borderRadius="3" bevel={hasBorderRadius}>
  <Box
    shadow={hasBorderRadius ? undefined : 'md'}
    background={background}
    padding={finalPadding}
    overflowX="hidden"
    overflowY="hidden"
  >
    {children}
  </Box>
</ShadowBevel>
```

Reply:
Scratch that! Just paired with @sarahill and we are including the shadow bevel effect with squared elements. I'm pushing an update that will allow you to accomplish this with `borderRadius` alone e.g.

```jsx
<ShadowBevel boxShadow="xs" borderRadius={hasBorderRadius ? '3' : '0-experimental'}>
  <Box
    background={background}
    padding={finalPadding}
    overflowX="hidden"
    overflowY="hidden"
  >
    {children}
  </Box>
</ShadowBevel>
```

Reply:
Researched/discussed offline and `none` is not a spec'd value for `border-radius`. The declaration accepts [`<length>` or `<percentage>` values](https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius#values) and it's `initial` value is `0`. We plan to leave this as is and update the `ShadowBevel` internals and mixins accordingly.